src/common/sorted-table/index.js

```diff
@@ -384,11 +414,15 @@ export default class SortedTable extends Component {
                   total: nVisibleItems
                 })}
                 <span className='pull-right'>
-                  {map(groupedActions, (btnProps, key) => <Action
-                    {...btnProps}
-                    key={key}
-                    handlerParam={selectedItems}
-                  />)}
+                  <ButtonGroup>
+                    {map(groupedActions, ({ icon, label, level, handler }, key) => <ActionRow
+                      btnStyle={level}
+                      handler={() => this._executeGroupedAction(handler)}
```

Comment:
Do not create a new function during each `render()`: take advantage of `handlerParam`.